src/rai_extensions/rai_grounding_dino/rai_grounding_dino/grounding_dino.py

```diff
@@ -19,10 +19,10 @@
 from typing import TypedDict
 
 import rclpy
+from rai_grounding_dino.boxer import GDBoxer
 from rclpy.node import Node
 from sensor_msgs.msg import Image
 
-from rai_grounding_dino.boxer import GDBoxer
```

Comment:
No idea why pre-commit decided to change that

Reply:
isort [ra] < [rc] probably, since it will consider both imports to be of the same category -- isort is not aware of the package name being formatted